lib/src/mirror_loader.dart

```diff
@@ -8,8 +8,8 @@ import 'dart:mirrors';
 import 'package:path/path.dart' as path;
 import 'package:initialize/initialize.dart';
 
-final _root = currentMirrorSystem().isolate.rootLibrary;
-final _libs = currentMirrorSystem().libraries;
+final LibraryMirror _root = currentMirrorSystem().isolate.rootLibrary;
```

Comment:
These explicit types shouldn't be necessary, were these providing a strong mode error?

Reply:
I get:

```
  hint • The type of '_root' can't be inferred because of the use of the instance getter 'rootLibrary' at lib/src/mirror_loader.dart:12:45 • strong_mode_top_level_instance_getter
```

Reply:
Ah ok  ya I forgot top level things don't have quite as nice inference :(